cocos/platform/CCImage.cpp

```diff
@@ -2178,34 +2178,34 @@ bool Image::saveImageToPNG(const std::string& filePath, bool isToRGB)
         png_infop info_ptr;
         png_bytep *row_pointers;
         
-        auto* outStream = FileUtils::getInstance()->openFileStream(filePath, FileStream::Mode::WRITE);
+        auto outStream = FileUtils::getInstance()->openFileStream(filePath, FileStream::Mode::WRITE);
 
         CC_BREAK_IF(nullptr == outStream);
 
         png_ptr = png_create_write_struct(PNG_LIBPNG_VER_STRING, nullptr, nullptr, nullptr);
 
         if (nullptr == png_ptr)
         {
-            delete outStream;
+            outStream = nullptr;
             break;
         }
 
         info_ptr = png_create_info_struct(png_ptr);
         if (nullptr == info_ptr)
         {
-            delete outStream;
+            outStream = nullptr;
```

Comment:
I prefer use `outStream.reset` when I want release the memory explicitly.

Reply:
If that's the preferred way, then I'll update all instances of it to use `reset()`.